sdk/containerregistry/Azure.Containers.ContainerRegistry/src/Blob/BlobHelper.cs

```diff
@@ -32,9 +32,13 @@ internal static string ComputeDigest(Stream stream)
 
         internal static string ComputeDigest(BinaryData data)
         {
+#if NET6_0_OR_GREATER
```

Comment:
I believe we'll need to add net6.0 and net7.0 targets to the .csproj for this to ever take effect.